mappings/net/minecraft/scoreboard/ScoreboardCriterion.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_1313 net/minecraft/scoreboard/ScoreboardCriterion
-	FIELD field_26748 CRITERIA Ljava/util/Map;
```

Comment:
these are criteria, as mentioned in the minecraft wiki

Reply:
Go check upstream